.rubocop_todo.yml

```diff
@@ -1,6 +1,6 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2017-09-10 17:36:28 +0200 using RuboCop version 0.49.1.
+# on 2017-09-12 09:31:35 +0200 using RuboCop version 0.49.1.
```

Comment:
The generation of the todo file should be in 7acb5894b6e4bee42e30d07ac6c76452cc6d06b3, as you need it for enabling the cop there.